### DISCUSSION TITLE
请教下关于tree组件如何设置不同节点的icon图标

### DISCUSSION THREAD

**Discussion by ljcoder** [layui/layui#1769] [问答]
请教下各位大佬，tree的icon图标如何动态的进行设置？在官方文档当中未查询到相应的属性信息
![image](https://github.com/layui/layui/assets/1784865/b73ec84f-f76f-4dbd-bd3b-49c5a61bca74)

![image](https://github.com/layui/layui/assets/1784865/3a9af226-16c2-462f-bc7d-826be4581a2a)

**Comment by Sight-wcg** [comment]
tree 组件目前不支持该功能，暂无调整计划，建议用第三方 tree 替代

**Reply by ljcoder** [reply]
好的谢谢
